src/Handlers/BaseRedisHandler.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace CodeIgniter\Settings\Handlers;
+
+use CodeIgniter\Cache\Handlers\RedisHandler;
+use Config\Cache;
+
+/**
+ * Redis class extension
+ */
+class BaseRedisHandler extends RedisHandler
+{
+    public function __construct(Cache $config)
+    {
+        parent::__construct($config);
+        $this->initialize();
+    }
+
+    public function getRedis()
+    {
+        return $this->redis;
```

Comment:
Why do you need to expose the `$redis` property? Is this for testing only?

Reply:
No, I use redis property in getHydrate function to search with pattern.

$keys = $this->handler->getRedis()->scan($iterator, $pattern);